README.md

```diff
@@ -77,18 +77,12 @@ The REST Layer framework is composed of several sub-packages:
 
 ## Breaking Changes
 
-Until version 1.0 of rest-layer, breaking changes may occur at any time if you rely on the latest master version.
-
-Below is an overview over recent breaking changes, starting from an arbitrary point with PR #151:
+Until we reach a stable v1, there will be occasional breaking changes to the
+rest-layer APIs. Breaking will however not arrive at patch releases.
 
-- PR #151: `ValuesValidator FieldValidator` attribute in `schema.Dict` struct replaced by `Values Field`.
-- PR #179: `ValuesValidator FieldValidator` attribute in `schema.Array` struct replaced by `Values Field`.
-- PR #204: 
-  - Storage drivers need to accept pointer to `Expression` implementer in `query.Predicate`.
-  - `filter` parameters in sub-query will be validated for type match.
-  - `filter` parameters will be validated for type match only, instead of type & constrains.
+Breaking changes since the latest minor release (v0.2.0), will be listed here.
 
-From the next release and onwards (0.2), this list will summarize breaking changes done to master since the last release.
+For breaking changes arriving in v0.2.0, see the [Release Notes](https://github.com/rs/rest-layer/releases/tag/v0.2.0).
```

Comment:
PS! This link won't start working until the release is actually out. As mentioned, I move the list out for v0.2.0 in particular, mostly because it's incomplete.